src/user-agent.ts

```diff
@@ -4,4 +4,15 @@
  * user agent of sdk
  * @hidden
  */
-export const USER_AGENT = "sdk-js-2.0.0-alpha.18";
+import { isSSR } from "./index";
+
+const scriptTag = (document.currentScript as HTMLScriptElement | null)?.src;
+let suffix = "";
+const ssr = isSSR();
+if (scriptTag) {
+  const source = new URL(scriptTag).host;
+  suffix = ` (via ${source})`;
+} else if (ssr) {
+  suffix = " (via SSR)";
+}
+export const USER_AGENT = `sajari-sdk-js/2.0.0-alpha.18${suffix}`;
```

Comment:
As with my comments on https://github.com/sajari/sdk-js/pull/87 I think that we just join different parts of the UA with a space, so no need for `(`s and `via`.

Also, the code might possibly read better with something like (untested):

```
export const USER_AGENT = ["sajari-sdk-js/2.0.0-alpha.18", suffix].filter(Boolean).join(" ");
```

Reply:
I think in this case, given it's a domain rather than a user agent, we'd want to keep the `(via x)` as I mentioned on the ticket 👍

Reply:
@sampotts yep it's probably fine as is. Same for SSR: ` (via SSR)` or just ` SSR`? I'd probably say the second one since it's not so much "via" but also may not matter.

Reply:
Yeah true, just `SSR` is probably fine there - e.g. `sajari-sdk-js/2.0.0-alpha.18 SSR`. 

However, I know Wikipedia is not a great source but... 

> For this reason, most Web browsers use a User-Agent string value as follows:
> Mozilla/[version] ([system and browser information]) [platform] ([platform details]) [extensions]

All I'll say is in this case perhaps the server-side rendering is classed as either `system and browser` or `platform` information? So perhaps... 

`sajari-sdk-js/2.0.0-alpha.18 (SSR)`

🤷‍♂️

Reply:
Yeah `xxx (SSR)` sounds good to me.